projects/backend/controllers/render.ts

```diff
@@ -252,25 +279,29 @@ export const renderFrontController = async (req: Request, res: Response) => {
         return
     }
 
-    const idFurniturePair = front.collections
+    const groupIdToFurnitureAndTheme = front.collections
         .map(collection =>
-            collection.items.map((item): [number, PublishedFurniture] => [
-                item.internalPageCode,
-                item.furniture,
-            ]),
+            collection.items.map((item): [
+                number,
+                PublishedFurniture,
+                Theme,
+            ] => {
+                const mappedTheme = mapSwatchToTheme(front.swatch)
```

Comment:
Thinking about it - does this even need to be inside this function? Ie we could leave that as it was an just do:

```
                const mappedTheme = mapSwatchToTheme(front.swatch)

```

and pass `mappedTheme` into `processArticleRendering` on line 304?

Reply:
Good point, I'll move this out!